.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I seem this file is a leftover you've forgotten to clean up, same with `.yarn/releases/yarn-1.22.5.cjs`.

Reply:
I have fixed it.